authority/voting/server/state.go

```diff
@@ -407,11 +428,9 @@ func (s *SharedRandom) Reveal() []byte {
 }
 
 func (s *state) reveal(epoch uint64) []byte {
-	if reveal, ok := s.reveals[epoch][s.identityPubKey()]; ok {
+	if reveal, ok := s.reveals[epoch][s.identityPubKeyHash()]; ok {
```

Comment:
wait why don't we just do this when we populate `s.reveals[epoch][s.identityPubKeyhash()] = srv.Reveal()` ?
Anyway, patch here looks fine.